File: submissions/sm_001_aalind/week_03/day_4/session_1/mirrorimage.js
// Print the mirror image of the given string AyzB should become ZbaY

function mirrorImage(word) {
  var cap_alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
  var small_alpha = "abcdefghijklmnopqrstuvwxyz";
  var new_word = "";

  for(var i = 0; i < word.length; i++) {
    var index = cap_alpha.indexOf(word[i]);

    if(index >= 0) {
      new_word += cap_alpha[25-index];
    } else {
      index = small_alpha.indexOf(word[i]);
      new_word += small_alpha[25-index];
    }
  }

  console.log(new_word);
}

mirrorImage("AyPzB");

File: submissions/sm_102_amit/week_13/day_4/session_1/mirror_image.py
# Print the mirror image of the given string AyzB should become ZbaY
string = input("Enter string: ")
mirror = ""

for s in string:
    v = ord(s)
    if v >= 65 and v <= 90:
        mv = 90 - (v - 65)
        mirror += chr(mv)
    elif v >= 97 and v <= 122:
        mv = 122 - (v - 97)
        mirror += chr(mv)
print(mirror)
